plugins/mtd/fu-self-test.c

```diff
@@ -32,7 +32,7 @@ fu_test_mtd_device_func(void)
 	ret = fu_context_load_quirks(ctx, FU_QUIRKS_LOAD_FLAG_NO_CACHE, &error);
 	g_assert_no_error(error);
 	g_assert_true(ret);
-	ret = fu_context_load_hwinfo(ctx, &error);
+	ret = fu_context_load_hwinfo(ctx, FU_CONTEXT_HWID_FLAG_NONE, &error);
```

Comment:
In the self tests don't we actually want it to match our expected environment?   Or is your intention that our environment is "empty"?

Reply:
mtd doesn't care about any specific hwids -- so i figured we should just do as little as possible.

Reply:
But that's how SMBIOS tables get loaded, and how we fixed the recent error on s390x because it didn't have them.   So perhaps we should back some of that out now?